tfx/orchestration/portable/base_executor_operator.py

```diff
@@ -84,6 +83,28 @@ def with_execution_watcher(
     self._execution_watcher_address = execution_watcher_address
     return self
 
+  @abc.abstractmethod
+  def handle_stop(self) -> None:
+    """Executor Operator specific logic to clean up after it is stopped."""
+    pass
+
+class BaseExecutorOperator(ParentBaseExecutorOperator):
```

Comment:
Is this really needed? I think it is sufficient to have `handle_stop` to be an abstractmethod, without needing to introduce `ParentBaseExecutorOperator`. I guess it is normal to have `handle_stop` implementation for any class that inherits `BaseExecutorOperator`.